server/local/studentdashboard-LOCALHOST.yml

```diff
@@ -7,6 +7,8 @@
 application_name: SD-QA
 # use file based data provider if not commented.
 data_provider_file_directory: server/test-files
+# used for testing response to throttled check url requests.
+#data_provider_file_uniqname: test-throttling-terms
```

Comment:
That comment is meant to preserve information to make testing easier down the road.